src/domain/checkForAndPlaceOrder.ts

```diff
@@ -109,32 +114,32 @@ async function _checkForAndPlaceOrder(
       const logPrefix = `[checkForAndPlaceOrder::${orderRef}]`;
       const logOrderDetails = `Processing order from TX ${conditionalOrder.tx} with params:`;
 
-      const { result: lastResult } = conditionalOrder.pollResult || {};
+      const { result: lastHint } = conditionalOrder.pollResult || {};
```

Comment:
Just curious, why `lastHint` for the `last poll result`?

Reply:
Sure, sorry I should have communicated my thinking behind this. It seems that the _outcome_ of the polling is to _hint_ to the watch-tower what should be done. This makes it easier, IMO, when reasoning.

IMO this then makes it easier when looking at comparisons, such as:

```typescript
      if (
        lastHint?.result === PollResultCode.TRY_AT_EPOCH &&
        blockTimestamp < lastHint.epoch
      ) {
```

Having some "meaning" that's built into the name, as opposed to contextual, I think makes easier reading here?

Reply:
okok, it also works